src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -71,23 +77,10 @@ void createStation() {
     @Test
     void createStationWithDuplicateName() {
         // given
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        RestAssured.given().log().all()
-                .body(params)
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when().post("/stations")
-                .then().log().all();
+        String name = createNewStation("강남역").body().jsonPath().get("name");
```

Comment:
```suggestion
        String name = 지하철_역_생성("강남역");
```

이렇게 해볼수도 있겠네요 :smile: